code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -1,3 +1,7 @@
+#define THEME_WIZARD "wizard"
+#define THEME_CULT "cult"
+#define THEME_HOLY "holy"
```

Comment:
Don't duplicate these defines, you lose the benefit of being able to just change the define and have everything work--now we have to update two places. Put them in a file in `__DEFINES`.

Reply:
Actually, I can't tell, do these have to be the same values as the one in soulstone.dm?

Reply:
They don't.